src/main/java/io/github/cadiboo/optifinedeobf/DeobfFrame.java

```diff
@@ -287,7 +287,8 @@ public synchronized void drop(DropTargetDropEvent evt) {
 	private void chooseFile(String title, Consumer<File> handler, JFileChooser chooser, FileFilter filter) {
 		chooser.setDialogTitle(title);
 		chooser.setFileSelectionMode(JFileChooser.FILES_ONLY);
-		chooser.setAcceptAllFileFilterUsed(false);
+		chooser.setAcceptAllFileFilterUsed(true);
+		chooser.addChoosableFileFilter(filter);
```

Comment:
What was broken that this fixes?  I've been using this on mac for years with no trouble, are you on windows by any chance?